docker/mockserv.Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM --platform=$BUILDPLATFORM golang:1.17-alpine AS build-env
+FROM --platform=$BUILDPLATFORM golang:1.19-alpine AS build-env
```

Comment:
Shouldn't we consistently use the same go version here as in go.mod?

Reply:
[go directive](https://go.dev/doc/modules/gomod-ref#go) from go.mod is defined as: 
> The minimum version of Go required to compile packages in this module.

I'm not sure what is the actual minimal version TBH (will check this out). Code-wise I don't think there are any cases prohibiting older Go versions.

I used latest version for CI, just because it's latest available.